### PR TITLE
#163856785  Change the request body and the url of login/signup feature according to the readme

### PR DESCRIPTION
**Pull request by oesukam**
#### What does this PR do?
Change the request body and the url of login/signup feature according to the readme

#### Description of Task to be completed?
- Change the request body of login/signup feature to reflect the readme
- Change the signup endpoint from `POST /api/v1/users/signup` to `POST /api/v1/users`

#### How should this be manually tested?
- git clone this branch

- run `npm install`

- run `npm migrate:seed`

- run `npm run dev`

- with postman use this link to signup:
`POST: api/v1/users/`

- Body request example (login):
```request-json
{
   "user": {
       "username": "username",
       "email": "username@mail.com"
       "password": "123456"
   }
}
```
- with postman use this link to login:
`POST: api/v1/users/login`

- Body request example (login):
```
{
    "user": {
         "username": "username",
	 "password": "123456"
    }
}
```
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
`#163856785`

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by mcaleb808 — COMMENTED** [review]
I reviewed, everything looks good except the response body without a message.

**Comment by kimotho-njoki** [comment]
So what is your ASK @mcaleb808 ?  What would you like added and on which line? Comment on the specific line

File: controllers/AuthController.js [hunk 1]
```diff
@@ -20,32 +20,31 @@ class AuthController {
...
-      return res.status(401).json({ status: 401, message: 'Please try again' });
+      return res.status(401).json({ message: 'Please try again' });
     }
 
     res.cookie('jwt', jwt, { httpOnly: true, secure: true });
-    const { password, ...userData } = user.get();
+    const { password, ...userData } = userModel.get();
     return res.status(201).json({
-      status: 201,
-      message: 'Account created sucessfully',
```
Comment (mcaleb808) [inline controllers/AuthController.js]: The message and status were removed because they don't appear in ReadMe signup response body.

Comment (kimotho-njoki) [reply controllers/AuthController.js]: Thank you for noting that @mcaleb808 .

So please remember that we are using Alphas examples on the README as examples, to guide us on how we are meant to format our work. But if we find that adding a status code and message is important in a particular response then we do so.

In this response, a status code, as well as message, are necessary. So please add them back.

File: controllers/AuthController.js [hunk 2]
```diff
@@ -72,7 +74,7 @@ class AuthController {
 
           res.cookie('jwt', jwt, { httpOnly: true, secure: true });
 
-          return res.json({ user: { ...user, token } });
+          return res.json({ status: 200, user: { ...user, token } });
```
Comment (mcaleb808) [inline controllers/AuthController.js:77]: Checked the file all status code and messages added

**Comment by oesukam** [comment]
@mcaleb808 @filschristian Please notify if you have already reviewed this PR.

**Review by filschristian — COMMENTED** [review]
The changes are okay. It is ready to be merged >>> @kimotho-njoki

**Review by filschristian — APPROVED** [review]
Already reviewed and it can be merged

**Review by mcaleb808 — APPROVED** [review]
I Caleb, I approved this pull Request to be merged.

**Pull request merged**